harvard-lts/drs-data-dashboard
# Adding hello world.

**Adding hello world.*
* * *

**JIRA Ticket**: [LIBDRS-7895](https://jira.huit.harvard.edu/browse/LIBDRS-7895)

# What does this Pull Request do?
This creates generic API Endpoints and a Hello-World HTML page to use as a jumping-off point for the DRS Data Dashboard. This was based off of the [Docker Flask Template](https://github.com/harvard-lts/docker-flask-template).

# How should this be tested?

* Completely rebuild the container: `docker-compose -f docker-compose-local.yml up -d --build --force-recreate`
* Confirm that you can go to https://localhost:3001/ You should see a generic API page for DRS Data Dashboard.
* Confirm that you can go to https://localhost:3001/version You should see a 0.0.1
* Confirm that you can go to https://localhost:3001/healthcheck You should see a successful healthcheck.
* Confirm that you can go to https://localhost:3001/hello-world You should see a Hello World HTML page.
* run `docker ps` you should see the container running as `drs-data-dashboard`
* Look in the /logs/ folder. You should see `drs-data-dashboard.log`, `supervisord_drs-data-dashboard_stderr.log` and `supervisord_drs-data-dashboard_stdout.log`
* run `docker-compose -f docker-compose.yml build` you should see it build a Docker image called `registry.lts.harvard.edu/lts/drs-data-dashboard   0.0.1 `


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz @dl-maura @mferrarini
